core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -260,7 +260,7 @@ class CoarseGrainedSchedulerBackendSuite extends SparkFunSuite with LocalSparkCo
 
     // To avoid allocating any resources immediately after releasing the resource from the task to
     // make sure that `availableAddrs` below won't change
-    when(ts.resourceOffers(any[IndexedSeq[WorkerOffer]])).thenReturn(Seq.empty)
+    when(ts.resourceOffers(any[IndexedSeq[WorkerOffer]], any[Boolean])).thenReturn(Seq.empty)
```

Comment:
This is added recently by another PR.

Reply:
Does it just affect master? OK in any event if it fixes a test.